### PR TITLE
HZN-657 : Write a Minion feature definition for opennms-poller-monitors-core

### PR DESCRIPTION
**Pull request by DP044946**

### CODE REVIEW

File: container/features/src/main/resources/features-minion.xml [hunk 1]
```diff
@@ -28,4 +28,10 @@
       <feature>minion-core-api</feature>
       <bundle>blueprint:mvn:org.opennms.features/org.opennms.features.discovery/${project.version}/xml/blueprint-discovery-distPollerDaoMinion</bundle>
     </feature>
+    
+    <feature name="opennms-poller-monitors-core" description="OpenNMS :: Poller :: Monitors :: Core" version="${project.version}">
```
Comment (soleger) [inline container/features/src/main/resources/features-minion.xml]: The name of this feature cannot be the same as the other poller-monitors-core feature. Please rename it to "opennms-poller-monitors-core-minion".

File: opennms-full-assembly/src/test/java/org/opennms/assemblies/karaf/MinionFeatureKarafIT.java [hunk 2]
```diff
@@ -75,4 +75,10 @@ public void testInstallFeatureOpennmsDiscoveryDistPollerDaoMinion() {
 		installFeature("opennms-discovery-distPollerDaoMinion");
 		System.out.println(executeCommand("features:list -i"));
 	}
+	
+	@Test
+	public void testInstallFeatureOpennmsPollerMonitersCore() {
+		installFeature("opennms-poller-monitors-core");
```
Comment (soleger) [inline opennms-full-assembly/src/test/java/org/opennms/assemblies/karaf/MinionFeatureKarafIT.java]: Rename this to match the new feature name above.

File: opennms-full-assembly/src/test/java/org/opennms/assemblies/karaf/OnmsFeatureKarafIT.java [hunk 3]
```diff
@@ -435,4 +435,10 @@ public void testInstallFeatureSpringWebflow() {
 		installFeature("spring-webflow");
 		System.out.println(executeCommand("features:list -i"));
 	}
+	
+	@Test
```
Comment (soleger) [inline opennms-full-assembly/src/test/java/org/opennms/assemblies/karaf/OnmsFeatureKarafIT.java]: Remove this test. Pradeep already added the test to this file.

**Comment by soleger** [comment]
Please merge feature-refactor-poller-monitors back into your branch. Some of the changes here have already been made in the main branch.
Deepak : done

File: opennms-full-assembly/src/test/java/org/opennms/assemblies/karaf/OnmsFeatureKarafIT.java (hunk 3 above)
Comment (DP044946) [inline opennms-full-assembly/src/test/java/org/opennms/assemblies/karaf/OnmsFeatureKarafIT.java]: Removed

File: features/poller/monitors/core/blueprint-poller-monitors-core-minion.xml [hunk 4]
```diff
@@ -0,0 +1,301 @@
+<blueprint xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0"
```
Comment (DP044946) [inline features/poller/monitors/core/blueprint-poller-monitors-core-minion.xml:1]: I just added this file to make my build successful,  this file is already merged to main branch from HZN-656.

File: opennms-full-assembly/src/test/java/org/opennms/assemblies/karaf/MinionFeatureKarafIT.java (hunk 2 above)
Comment (DP044946) [inline opennms-full-assembly/src/test/java/org/opennms/assemblies/karaf/MinionFeatureKarafIT.java]: Completed & pushed the changes

File: container/features/src/main/resources/features-minion.xml (hunk 1 above)
Comment (DP044946) [inline container/features/src/main/resources/features-minion.xml]: Renamed it to opennms-poller-monitors-core-minion & pushed the changes

**Comment by soleger** [comment]
These changes have been merged into the feature branch. Closing PR.

**Pull request closed without merging**